### PR TITLE
Add suffix '.slice' with `systemd` cgroup driver on reserved cgroup name.

### PR DESCRIPTION
**Pull request by tossmilestone**
Add suffix '.slice' with `systemd` cgroup driver on reserved cgroup name.

Close kubernetes/kubernetes#110288

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/reserve-compute-resources.md [hunk 1]
```diff
@@ -95,7 +95,9 @@ system daemon should ideally run within its own child control group. Refer to
 for more details on recommended control group hierarchy.
 
 Note that Kubelet **does not** create `--kube-reserved-cgroup` if it doesn't
-exist. Kubelet will fail if an invalid cgroup is specified.
+exist. Kubelet will fail if an invalid cgroup is specified. With `systemd`
+cgroup driver, you should create the control group with value of
+`--kube-reserved-cgroup` subfixed with `.slice`.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/reserve-compute-resources.md:100]: Adapting this to be idiomatic English, but keeping the meaning I think you intended:
```suggestion
cgroup driver, you should follow a specific pattern for the name of the cgroup you
define: the name should be the value you set for `--kube-reserved-cgroup`,
with `.slice` appended.
```

Note that “subfixed” is not a word in English (you might have meant suffixed).

File: content/en/docs/tasks/administer-cluster/reserve-compute-resources.md [hunk 2]
```diff
@@ -120,7 +122,9 @@ It is recommended that the OS system daemons are placed under a top level
 control group (`system.slice` on systemd machines for example).
 
 Note that `kubelet` **does not** create `--system-reserved-cgroup` if it doesn't
-exist. `kubelet` will fail if an invalid cgroup is specified.
+exist. `kubelet` will fail if an invalid cgroup is specified.  With `systemd`
+cgroup driver, you should create the control group with value of
+`--system-reserved-cgroup` subfixed with `.slice`.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/reserve-compute-resources.md:127]: Adapting this to be idiomatic English, but keeping the meaning I think you intended:
```suggestion
cgroup driver, you should follow a specific pattern for the name of the cgroup you
define: the name should be the value you set for `--system-reserved-cgroup`,
with `.slice` appended.
```

Note that “subfixed” is not a word in English (you might have meant suffixed).

**Review by sftim — DISMISSED** [review]
/sig node

This is a “fix” for a support issue awaiting triage. For me, that's a concern - we should mostly aim to fix issues that are triaged as accepted.
I'd prefer to see a [new issue](https://github.com/kubernetes/website/issues/new/choose) tracking the improvement we're making here, and to have SIG Node confirm that this is a sound recommendation to make.

@BenTheElder's [comment](https://github.com/kubernetes/kubernetes/issues/110288#issuecomment-1145554369) is relevant, but to me it doesn't serve as enough confirmation that this is the right recommendation here. 

https://blog.kintone.io/entry/2022/03/08/170206 doesn't mention reserved cgroups either. Maybe there's an opportunity for someone to write a blog article about this aspect.

**Comment by tossmilestone** [comment]
@sftim updated, PTAL, thanks!

**Comment by sftim** [comment]
Thanks @tossmilestone @dashpole 

/approve

**Pull request merged**